tests/requirements-testing.txt

```diff
@@ -1,5 +1,6 @@
 coverage==6.3
 dirty-equals==0.1
+redislite==6.2.805324
```

Comment:
I don't think we need this, if we do, it's a different discussion and PR.

Reply:
This is only for testing. I haven't found any easy way to include on the CI workflow a redis-server that is listening on a unix socket.

Reply:
Ok, that's fine then.